### PR TITLE
Announcement blog post for 4.0.0.pre3

### PR DESCRIPTION
**Pull request by jasonrclark**
~~**Note, please don't merge until 4.0.0.pre3 is actually out!**~~

To go along with shoes/shoes4#1008, here's a blog post. Mostly focused on the gem changes.

### CODE REVIEW

File: _posts/2015-01-04-shoes-4pre3-release.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+title: "shoes 4 preview 3 released!"
+slug: "shoes-4-preview-3-released"
+date: 2015-01-04 09:25
+layout: default
+---
+
+<div class='jumbotron'>
+  <h1>The Shoes Blog</h1>
+</div>
+<h1><a href="{{ post.url }}">shoes 4 preview 3 released!</a></h1>
```
Comment (PragTob) [inline _posts/2015-01-04-shoes-4pre3-release.md]: We wanted to capitalize "Shoes" - I still always do it wrong :o

File: _posts/2015-01-04-shoes-4pre3-release.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<em>2015/01/04</em>
+
+Howdy Shoes fans!
+
+[A new pre-release version
+(4.0.0.pre3)](https://rubygems.org/gems/shoes/versions/4.0.0.pre2) is up on
```
Comment (PragTob) [inline _posts/2015-01-04-shoes-4pre3-release.md]: link is wrong :) should be pre3 as the target

**Comment by PragTob** [comment]
Looks good, thanks!

**Comment by jasonrclark** [comment]
Good catches. Updated!

**Comment by wasnotrice** [comment]
Very nice. :ship:

**Pull request merged**